Formula/minetest.rb

```diff
@@ -28,33 +33,73 @@ class Minetest < Formula
   head do
     url "https://github.com/minetest/minetest.git"
 
+    resource "irrlichtmt" do
+      url "https://github.com/minetest/irrlicht.git", branch: "master"
+    end
+
     resource "minetest_game" do
       url "https://github.com/minetest/minetest_game.git", branch: "master"
     end
   end
 
   depends_on "cmake" => :build
-  depends_on "freetype"
-  depends_on "gettext"
-  depends_on "irrlicht"
-  depends_on "jpeg"
-  depends_on "libogg"
-  depends_on "libvorbis"
-  depends_on "luajit"
+  depends_on "gettext" => :build
+
+  uses_from_macos "bzip2"
+  uses_from_macos "curl"
+  uses_from_macos "ncurses"
+  uses_from_macos "sqlite"
+  uses_from_macos "zlib"
+
+  on_macos do
+    # NOTE: The following dependencies are not used at runtime since the
+    # build packages copies of dylibs into minetest.app/Contents/Frameworks/
+    depends_on "freetype" => :build
+    depends_on "jpeg" => :build
+    depends_on "libogg" => :build
+    depends_on "libpng" => :build
+    depends_on "libvorbis" => :build
+    depends_on "luajit-openresty" => :build
+    depends_on "zstd" => :build
+  end
```

Comment:
We should replace those copies with symlinks if we can.

Reply:
For now, trying the easier fix of disabling `fixup_bundle`.

Otherwise, it would take a number of workarounds to relink as there are a number of dylibs and they are all major-versioned (with some like `luajit` and `zstd` having fully versioned libs copied).
```console
❯ ls /usr/local/Cellar/minetest/5.5.0/minetest.app/Contents/Frameworks
libfreetype.6.dylib  libjsoncpp.25.dylib        libogg.0.dylib     libvorbisfile.3.dylib
libgmp.10.dylib      libluajit-5.1.2.1.0.dylib  libpng16.16.dylib  libzstd.1.5.2.dylib
libjpeg.9.dylib      libluajit-5.1.2.dylib      libvorbis.0.dylib  libzstd.1.dylib
```

Reply:
```console
$ brew linkage minetest
System libraries:
  /usr/lib/libSystem.B.dylib
  /usr/lib/libc++.1.dylib
  /usr/lib/libcurl.4.dylib
  /usr/lib/libform.5.4.dylib
  /usr/lib/libiconv.2.dylib
  /usr/lib/libncurses.5.4.dylib
  /usr/lib/libobjc.A.dylib
  /usr/lib/libsqlite3.dylib
  /usr/lib/libz.1.dylib
Homebrew libraries:
  /usr/local/opt/freetype/lib/libfreetype.6.dylib (freetype)
  /usr/local/opt/gmp/lib/libgmp.10.dylib (gmp)
  /usr/local/opt/jpeg/lib/libjpeg.9.dylib (jpeg)
  /usr/local/opt/jsoncpp/lib/libjsoncpp.25.dylib (jsoncpp)
  /usr/local/opt/libogg/lib/libogg.0.dylib (libogg)
  /usr/local/opt/libpng/lib/libpng16.16.dylib (libpng)
  /usr/local/opt/libvorbis/lib/libvorbis.0.dylib (libvorbis)
  /usr/local/opt/libvorbis/lib/libvorbisfile.3.dylib (libvorbis)
  /usr/local/opt/luajit-openresty/lib/libluajit-5.1.2.dylib (luajit-openresty)
  /usr/local/opt/zstd/lib/libzstd.1.dylib (zstd)
```